metadata-models/src/main/pegasus/com/linkedin/metadata/search/DashboardDocument.pdl

```diff
@@ -41,5 +36,5 @@ record DashboardDocument includes BaseDocument {
   /**
    * Access level for the dashboard
    */
-  access: optional string
+  access: optional AccessLevel
```

Comment:
Can we leave model changes to their own PRs, outside of implementation? Also, we just had a round of design reviews for this, why did this change (blanket question that applies to other changes like removing chartIds)

Reply:
This is actually an innocent model change :) I earlier forgot that we support enums in the document models and set the types for a couple of enum fields to string. Now, setting them back to their actual types for better type checking.

Reply:
Regarding removing chartIds: I originally thought that we can store dashboard <-> chart <-> dataset (or metric) relationships in the search index to support queries like "what are the downstream dashboards of this dataset?". However, any kind of relationship query could be supported by graph (that's what it's for in the end) and think that no need to store these relations in the search index. WDYT?

Reply:
+1 on leaving the modeling changes to separate PRs. I am not an elasticsearch expert, but I can leave feedback on models and how they would impact our usage. Right now the PR is pretty large and it makes that more difficult.

Reply:
What does "innocent" mean? Unrelated? More of a reason to put in a different PR, imo.

Reply:
Sure, let me send model changes as a separate PR ;)

Reply:
New PR for model updates only: https://github.com/linkedin/datahub/pull/1932